app/example_user.rb

```diff
@@ -0,0 +1,26 @@
+class User
+    attr_accessor :first_name, :last_name
+    attr_accessor :name, :email
```

Comment:
we use 2-space indentation for both Ruby & JS, you should be able to set it in your text editor but let me know if you need a hand :)

Reply:
I've configured VSCode to do this, seems like it did not work will have a look at it again.